backend/src/main/java/org/latexscribe/LatexScribe/domain/model/Document.java

```diff
@@ -0,0 +1,35 @@
+package org.latexscribe.LatexScribe.domain.model;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+
+import java.time.LocalDateTime;
+
+@Entity
+@Getter
+@Setter
+@Table(name = "document")
+@RequiredArgsConstructor
+public class Document {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String filename;
+    private Long filesize;
+    @Column(name = "last_modified")
+    private LocalDateTime lastModified;
+    @Lob
+    @Column(name = "file_content")
+    private byte[] fileContent;
```

Comment:
```suggestion
    @Column(name = "filename")
    private String name;
    @Column(name = "filesize")
    private Long size;
    @Column(name = "last_modified")
    private LocalDateTime lastModified;
    @Lob
    @Column(name = "file_content")
    private byte[] content;
```

I think we should remove the file prefixes form the `Document` files, this should reduce the code noise, removing the redundant name `doucment.content` instead of `doucment.fileContent`.

We can rename the fields in the database with the `@Column` to keep the naming while simplifying the usages of the fields